modules/UI/videolayout/LargeVideoManager.js

```diff
@@ -90,7 +97,8 @@ export default class LargeVideoManager {
     destroy() {
         window.clearInterval(this._updateVideoResolutionInterval);
         this.videoContainer.removeResizeListener(
-            this._updateVideoResolutionStatus);
+            this._onVideoResolutionUpdate);
+        this._videoAspectRatio = null;
```

Comment:
Why are you setting the value to `null`? 

Why not `0` which is the initial value or `undefined`? Actually do we need to change it at all?